server/repository/src/migrations/v2_06_00/drop_legacy_reports.rs

```diff
@@ -0,0 +1,19 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "drop_legacy_reports"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        sql!(
+            connection,
+            r#"
+                DROP TABLE report;
```

Comment:
can you make it one migration, together with the other one, it's easier to understand the logic this way, and don't drop table, just delete everything from it

Reply:
Actually I noticed in another PR that we still have ReportType. I think it's ok to drop and then re-create but remove ReportType